specs/phase0/beacon-chain.md

```diff
@@ -563,6 +565,23 @@ class SignedBeaconBlockHeader(Container):
 
 *Note*: The definitions below are for specification purposes and are not necessarily optimal implementations.
 
+
+### Validation
+
+#### `require`
+
+```python
+def require(condition: bool, message: str=None) -> None:
```

Comment:
Strictly speaking, it should be `Optional[str]` in Python:

```suggestion
from typing import Optional 
def require(condition: bool, message: Optional[str]=None) -> None:
```

- Python `Optional` might be confusing for the readers. Although `Optional` has been used in validator.md and das-core.md anyway...
- mypy may raise an error if we use a stricter config

I'm fine either way for now.

Reply:
We could force inplicit optionals (https://mypy.readthedocs.io/en/stable/config_file.html#none-and-optional-handling) in the `linter.ini`